source/content/terminus/install.md

```diff
@@ -48,23 +48,14 @@ The following commands will:
   sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
   ```
 
-<Alert type="info" title="Note">
```

Comment:
The Terminus PHAR instructions above are now wrong. See https://github.com/pantheon-systems/terminus/issues/2270.

Reply:
This will work again once the Terminus 2 build finishes, and T2 is again the latest release.

Reply:
thanks for identifying this issue, @heddn 

we'll keep an eye on the conversation there

---

Docs:
re: [L46](https://github.com/pantheon-systems/documentation/pull/6786/files#diff-e9f62000044534930e5e88fb1f7a29420519da0db1d43067db19a161dc40f7aeR46)

potential fixed line is:

```shell
curl -L https://github.com/pantheon-systems/terminus/releases/download/`curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m#"tag_name": "\K[^"]*#g'`/terminus --output terminus
```

but this might be remediated by a [fix to Terminus](https://github.com/pantheon-systems/terminus/issues/2270#issuecomment-989170883) per @greg-1-anderson

Reply:
looks like this is now fixed and the line as documented works to install `3.0.1` 

```shell
❯ ./terminus self:info
 ------------------------- -------------------------------------------------------------
  PHP binary                /usr/local/Cellar/php/8.0.13/bin/php
  PHP version               8.0.13
  php.ini used              /usr/local/etc/php/8.0/php.ini
  Terminus project config
  Terminus root dir         phar:///Users/edwardangert/projects/delete/terminus
  Terminus version          3.0.1
  Operating system          Darwin Kernel Version 20.6.0: Tue Oct 12 18:33:42 PDT 2021;
                            root:xnu-7195.141.8~1/RELEASE_X86_64
 ------------------------- -------------------------------------------------------------
```